polaris-react/stylelint/plugins/coverage/configs/index.js

```diff
@@ -0,0 +1,127 @@
+module.exports = {
+  plugins: ['stylelint-scss', '../'],
+  rules: {
+    'at-rule-disallowed-list': ['keyframes'],
```

Comment:
I'm a little surprised to see this as an error. Is the idea that Polaris will or should expose keyframes via custom properties or components? Similar to [open-props/animations](https://github.com/argyleink/open-props/blob/main/src/props.animations.css)